src/main/kotlin/no/nav/familie/ba/mottak/hendelser/IdenthendelseConsumer.kt

```diff
@@ -0,0 +1,62 @@
+package no.nav.familie.ba.mottak.hendelser
+
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.Metrics
+import no.nav.familie.ba.mottak.task.SendIdenthendelseTilSakTask
+import no.nav.familie.kontrakter.felles.PersonIdent
+import no.nav.familie.prosessering.domene.TaskRepository
+import no.nav.person.pdl.aktor.v2.Aktor
+import no.nav.person.pdl.aktor.v2.Type
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.kafka.annotation.KafkaListener
+import org.springframework.kafka.support.Acknowledgment
+import org.springframework.stereotype.Service
+import javax.transaction.Transactional
+
+@Service
+@ConditionalOnProperty(
+    value = ["funksjonsbrytere.kafka.enabled"],
+    havingValue = "true",
+    matchIfMissing = true
+)
+class IdenthendelseConsumer(private val taskRepository: TaskRepository) {
+
+    val identhendelseFeiletCounter: Counter = Metrics.counter("barnetrygd.hendelse.ident.feilet")
+
+    @KafkaListener(
+        topics = ["aapen-person-pdl-aktor-v1"],
+        id = "personhendelse",
+        idIsGroup = false,
+        containerFactory = "kafkaIdenthendelseListenerContainerFactory"
+    )
+    @Transactional
+    fun listen(consumerRecord: ConsumerRecord<String, Aktor>, ack: Acknowledgment) {
+        val aktør = consumerRecord.value()
+        val folkeregisterident = aktør.identifikatorer.single { it.type == Type.FOLKEREGISTERIDENT && it.gjeldende }
+
+        try {
+            SECURE_LOGGER.info("Har mottatt ident-hendelse $consumerRecord")
+            if (opprettTask) {
+                val task =
+                    SendIdenthendelseTilSakTask.opprettTask(ident = PersonIdent(ident = folkeregisterident.idnummer.toString()))
```

Comment:
Trenger man task? Jeg er bekymret over at ba-mottak vil oversømmes av tasker med det pattern dere har valgt

Reply:
Så lenge vi bruker `latest`-strategi på offset så vil jeg ikke tro det. Men om vi skulle ha behov for å kjøre `earliest` på et tidspunkt så kan det nok bli litt i overkant ja.

Reply:
Samtidig synes jeg tasken gir lite verdi når den kun gjør innsending til sak. Hvis endepunkt mot sak går ned, så vil alle tasker feile uansett. Da kunne like gjerne feilet i listeneren.

Reply:
Da må jeg bare spørre dumt: Vil da meldingen bli forsøkt konsumert igjen senere (siden vi vel ikke får ack-et dersom det feiler)? I så fall er jeg enig i at tasken kan fjernes. Hele poenget var å ha muligheten til å rekjøre identhendelsen.

Reply:
ja, og kafkaErrorHandler vil stoppe og restarte listener for topicen. Først 2 ganger ganske kjapt og så hver 3 time.

Reply:
Stopper og restarter for å hindre masselogging.